packages/catlog/src/one/path.rs

```diff
@@ -48,6 +48,12 @@ pub enum Path<V, E> {
     Seq(NonEmpty<E>),
 }
 
+impl<V, E> From<E> for Path<V, E> {
+    fn from(e: E) -> Self {
+        Path::single(e)
+    }
+}
```

Comment:
Possibly confusing in the common case where `V = E`, say both are UUIDs. When you convert into `Path`, do you get `Path::Id` or `Path::single`?

Reply:
Oh, I see, that's a subtle bug. I've addressed everything else, let me think about this one a bit.

Reply:
I think I was assuming from would be called on an morphism generator. Maybe there's a better way to do this though.

Reply:
E.g., have this be a method on `FgCategory` with some descriptive name like "mor_from_mor_gen". What would be a good name for this?

Reply:
On further discussion, we agreed that this is fine because paths are morphisms, so it makes sense to have the standard conversion be from edges to paths.